File: demo/uniapp-demo/socialuni/model/talk/TalkQueryVO.ts
import { locationModule } from '/socialuni/store'

export default class TalkQueryVO {
  public talkIds: number[]
  public adCode: string
  public lon: number
  public lat: number
  public tagIds: number[]
  public homeTabType: string
  public gender: string
  public minAge: number
  public maxAge: number

  constructor (talkIds: number[], tagIds: number[], tabType: string, gender: string, minAge: number, maxAge: number) {
    this.talkIds = talkIds
    const district = locationModule.location
    if (district) {
      this.adCode = district.adCode
      this.lon = district.lon
      this.lat = district.lat
    }
    this.tagIds = tagIds
    this.homeTabType = tabType
    this.gender = gender
    this.minAge = minAge
    this.maxAge = maxAge
  }
}

File: socialuni-center-web/src/main/java/com/socialuni/center/web/model/VO/TalkQueryVO.java
package com.socialuni.center.web.model.VO;

import lombok.Data;

import java.util.List;

/**
 * @author qinkaiyuan
 * @date 2019-08-21 20:47
 */
@Data
public class TalkQueryVO {
    private List<String> talkIds;
    private String homeType;
    private String tabType;
    private List<Integer> tagIds;

    private String adCode;
    private Double lon;
    private Double lat;

    private Integer minAge;
    private Integer maxAge;
    private String gender;
    //新版本用genderType，旧版本用gender
    private String genderType;
    private Boolean openPosition;
    //平台
    private String platform;
    //备用属性
    private String standby;
}
